cmd/faucet/faucet.go

```diff
@@ -613,24 +615,33 @@ func (f *faucet) loop() {
 				continue
 			}
 			// Faucet state retrieved, update locally and send to clients
-			f.lock.RLock()
 			log.Info("Updated faucet state", "number", head.Number, "hash", head.Hash(), "age", common.PrettyAge(timestamp), "balance", f.balance, "nonce", f.nonce, "price", f.price)
-
-			balance := new(big.Int).Div(f.balance, ether)
-			peers := f.stack.Server().PeerCount()
-
+			dataPack := map[string]interface{}{
+				"funds":    new(big.Int).Div(f.balance, ether),
+				"funded":   f.nonce,
+				"peers":    f.stack.Server().PeerCount(),
+				"requests": make([]*request, 0),
```

Comment:
I'm not familiar with golang web backend. If the requests are sent as empty, will the last sent data be rendered? Or the empty slice will be rendered.

Reply:
It's here: https://github.com/ethereum/go-ethereum/blob/master/cmd/faucet/faucet.html#L156 . Yeah actually this will probably not fly

Reply:
Reverted